pkg/config/setup/config.go

```diff
@@ -705,6 +705,8 @@ func InitConfig(config pkgconfigmodel.Config) {
 
 	// Admission controller
 	config.BindEnvAndSetDefault("admission_controller.enabled", false)
+	config.BindEnvAndSetDefault("admission_controller.validation.enabled", true)
+	config.BindEnvAndSetDefault("admission_controller.mutation.enabled", true)
```

Comment:
Should these new config parameters also be documented in the default config file:
https://github.com/DataDog/datadog-agent/blob/1a98d3866b7b0da29325ebb06563ee1653ac6900/pkg/config/config_template.yaml#L2900-L3107
?

Reply:
Yes! I'll add this to a subsequent PR with the documentation of the webhooks mentioned by David 👍

Reply:
Finally I ended up doing it now in this commit: https://github.com/DataDog/datadog-agent/pull/28512/commits/a873e650761aa86128333c4b16cdf2ace46242ea